projects/plugins/jetpack/class.jetpack.php

```diff
@@ -5008,6 +5009,25 @@ public static function authorize_starting() {
 		Jetpack_Options::update_option( 'unique_connection', $jetpack_unique_connection );
 	}
 
+	/**
+	 * This action fires when the site is registered (connected at a site level).
+	 */
+	public function handle_unique_registrations_stats() {
```

Comment:
What's the thought on adding this to the Jetpack plugin vs the Connection package?

Reply:
Just because its counterpart `unique_connection` is here: https://github.com/Automattic/jetpack/blob/f8969425fdf4fe2aa08a797ea86d321238436ba4/projects/plugins/jetpack/class.jetpack.php#L5003

I thought we could move them together in a following PR, but maybe we could do this right away